core/arch/aarch64/aarch64.asm

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
I would file a separate dedicated bug on the signal test failures we are seeing: I think that will be easier to understand and track this fix and have something to point at for what is blocking us.

Reply:
Filed #6758.